PendingReleaseNotes.md

```diff
@@ -11,6 +11,7 @@
 - A new CR property `skipUpgradeChecks` has been added, which allows you force an upgrade by skipping daemon checks. Use this at **YOUR OWN RISK**, only if you know what you're doing. To understand Rook's upgrade process of Ceph, read the [upgrade doc](Documentation/ceph-upgrade.html#ceph-version-upgrades).
 - Mon Quorum Disaster Recovery guide has been updated to work with the latest Rook and Ceph release.
 - A new CRD property `PreservePoolsOnDelete` has been added to Filesystem(fs) and Object Store(os) resources in order to increase protection against data loss. if it is set to `true`, associated pools won't be deleted when the main resource(fs/os) is deleted. Creating again the deleted fs/os with the same name will reuse the preserved pools.
+- The `status.Phase` field has been added to all the Ceph related CRs.
```

Comment:
Can you be more descriptive here? What's the information provided by this field?

Reply:
done